### PR TITLE
fix(bucket): generate bucket guide requests safely

### PR DESCRIPTION
**Pull request by htuna07**

### CODE REVIEW

File: stacks/spica/src/bucket/pages/index/index.component.html [hunk 1]
```diff
@@ -174,7 +174,7 @@ <h5>
             mat-resize-header
             [persist-header-width]="schema._id"
           >
-            id
+            _id
```
Comment (thesayyn) [inline stacks/spica/src/bucket/pages/index/index.component.html:177]:
```suggestion
            id
```

The end-user does not need to know that this field has leading underscore. Can we revert this change?

File: stacks/spica/src/bucket/pages/index/index.component.ts [hunk 2]
```diff
@@ -83,7 +83,7 @@ export class IndexComponent implements OnInit {
         this.guideResponse = {};
         this.readOnly = schema.readOnly;
         this.properties = [
-          {name: "$$spicainternal_id", title: "id"},
+          {name: "$$spicainternal_id", title: "_id"},
```
Comment (thesayyn) [inline stacks/spica/src/bucket/pages/index/index.component.ts:86]: Same as above

**Review by thesayyn — CHANGES_REQUESTED** [review]

File: stacks/spica/src/bucket/pages/index/index.component.html (hunk 1 above)
Comment (htuna07) [reply stacks/spica/src/bucket/pages/index/index.component.html:177]: User can create a field  named as **id** but not **_id**. If user creates **id** field there will be 2 id field and it will be more confusing. Also guide responses include **_id** instead of **id**. I think these changes will make things more understandable for users.

Comment (thesayyn) [reply stacks/spica/src/bucket/pages/index/index.component.html:177]: Yes that makes sense

**Review by thesayyn — APPROVED** [review]
